### DISCUSSION TITLE
Node 14 support

### DISCUSSION THREAD

**Discussion by Mister-Hope** [gulpjs/gulp#2598] [Help]
Node 14 is already in LTS, but `watch` command is not working well after I switched to Node 14.

The [related issue](https://github.com/gulpjs/glob-watcher/issues/55) is locked.

Also I found that this repo refuses to update any deps and are still trying to support node versions back to 0.10. And the last update is more than 2 years ago.

So will this project support newer versions of Node? Or it's just a historical tools for poeple using old versions?

Do we have other alternatives if we want similiar tools on Node 14? Thanks!

**Comment by yocontra** [comment]
@Mister-Hope You can follow progress on this here: https://github.com/gulpjs/glob-watcher/pull/61

**Reply by sryze** [reply]
what about v5? is it not happening?
